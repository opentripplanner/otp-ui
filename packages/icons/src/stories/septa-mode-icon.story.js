import React from "react";

import SeptaModeIcon from "../septa-icons";

import ModeIconRenderer from "./mode-icon-renderer";

export default {
  title: "Icons/SeptaModeIcon",
  component: SeptaModeIcon
};

export const SeptaModeIconExamples = () => (
  <ModeIconRenderer component={SeptaModeIcon} />
);
