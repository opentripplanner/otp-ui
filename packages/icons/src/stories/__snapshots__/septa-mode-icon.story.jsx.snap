// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons/SeptaModeIcon SeptaModeIconExamples smoke-test 1`] = `
<table class="icon-renderer__StyledTable-sc-g7krq-0 gVtNli">
  <thead>
    <tr>
      <th>
        Mode Type
      </th>
      <th>
        Result
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        BICYCLE
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="20.469"
               height="30"
               viewbox="0 0 21 15"
          >
            <path id="Bike"
                  d="M871.138,64.5a4.216,4.216,0,0,0-.761.068l-1.32-3.617v-1.99h.705a.654.654,0,1,0,0-1.308h-2.73a.654.654,0,1,0,0,1.308h.717V60.4h-5.539V58.315a.66.66,0,0,0-.659-.66h-2.741a.654.654,0,0,0,0,1.308H860.9v1.99l-1.319,3.617a4.357,4.357,0,0,0-.773-.068,4.079,4.079,0,1,0,2,.523l.819-2.24L864.36,68.8l.023.045a.606.606,0,0,0,.341.33.184.184,0,0,0,.1.034.326.326,0,0,0,.148.023h2.138a4.075,4.075,0,1,0,4.026-4.731Zm-9.552,4.083a2.769,2.769,0,1,1-2.775-2.775,2.8,2.8,0,0,1,.318.023l-.921,2.525a.653.653,0,1,0,1.229.443l.92-2.536A2.813,2.813,0,0,1,861.586,68.584Zm8.017-2.32.6,1.66h-1.762A2.815,2.815,0,0,1,869.6,66.264ZM862.576,61.7h4.811L864.975,67Zm4.537,6.22h-1.126l2.342-5.152.819,2.24A4.067,4.067,0,0,0,867.113,67.924Zm4.026,3.423a2.769,2.769,0,0,1-2.695-2.115h2.695a.471.471,0,0,0,.182-.035h.011a.11.11,0,0,0,.057-.023.514.514,0,0,0,.3-.261.6.6,0,0,0,.1-.341.578.578,0,0,0-.034-.193c0-.023-.011-.034-.011-.057l-.91-2.5a2.772,2.772,0,1,1,.307,5.527Z"
                  transform="translate(-854.74 -57.655)"
            >
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        BICYCLE_RENT
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="20.469"
               height="30"
               viewbox="0 0 21 15"
          >
            <path id="Bike"
                  d="M871.138,64.5a4.216,4.216,0,0,0-.761.068l-1.32-3.617v-1.99h.705a.654.654,0,1,0,0-1.308h-2.73a.654.654,0,1,0,0,1.308h.717V60.4h-5.539V58.315a.66.66,0,0,0-.659-.66h-2.741a.654.654,0,0,0,0,1.308H860.9v1.99l-1.319,3.617a4.357,4.357,0,0,0-.773-.068,4.079,4.079,0,1,0,2,.523l.819-2.24L864.36,68.8l.023.045a.606.606,0,0,0,.341.33.184.184,0,0,0,.1.034.326.326,0,0,0,.148.023h2.138a4.075,4.075,0,1,0,4.026-4.731Zm-9.552,4.083a2.769,2.769,0,1,1-2.775-2.775,2.8,2.8,0,0,1,.318.023l-.921,2.525a.653.653,0,1,0,1.229.443l.92-2.536A2.813,2.813,0,0,1,861.586,68.584Zm8.017-2.32.6,1.66h-1.762A2.815,2.815,0,0,1,869.6,66.264ZM862.576,61.7h4.811L864.975,67Zm4.537,6.22h-1.126l2.342-5.152.819,2.24A4.067,4.067,0,0,0,867.113,67.924Zm4.026,3.423a2.769,2.769,0,0,1-2.695-2.115h2.695a.471.471,0,0,0,.182-.035h.011a.11.11,0,0,0,.057-.023.514.514,0,0,0,.3-.261.6.6,0,0,0,.1-.341.578.578,0,0,0-.034-.193c0-.023-.011-.034-.011-.057l-.91-2.5a2.772,2.772,0,1,1,.307,5.527Z"
                  transform="translate(-854.74 -57.655)"
            >
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        BUS
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="25.023"
               height="25.022"
               viewbox="0 0 25.023 25.022"
          >
            <g id="Group_57689"
               data-name="Group 57689"
               transform="translate(-845 -293)"
            >
              <path id="Path_36555"
                    data-name="Path 36555"
                    d="M34.988,32.7v1.031a.863.863,0,1,0,1.725,0c0-.263-.014-1.031-.014-1.031h1.14V26.585s-.313-3.1-.442-4.326c-.144-1.37-.461-1.77-1.424-2.224a10.613,10.613,0,0,0-4.562-.924,10.616,10.616,0,0,0-4.563.924c-.963.455-1.279.854-1.424,2.224-.13,1.23-.442,4.326-.442,4.326V32.7h1.14s-.014.768-.014,1.031a.863.863,0,1,0,1.725,0V32.7Zm-8.874-2.88a.859.859,0,1,1,.859.859.859.859,0,0,1-.859-.859m8.879,0a.859.859,0,1,1,.859.859.859.859,0,0,1-.859-.859m-6.234-8.762a.392.392,0,1,1,0-.783H34.1a.392.392,0,1,1,0,.783Zm-2.047,5.577A.577.577,0,0,1,26.127,26c.1-1.375.181-2.151.349-3.531a.612.612,0,0,1,.671-.506H35.69a.629.629,0,0,1,.671.529c.186,1.453.25,2.093.353,3.5a.554.554,0,0,1-.551.638Z"
                    transform="translate(826.1 278.66)"
                    fill="#1a1818"
              >
              </path>
              <g id="Ellipse_528"
                 data-name="Ellipse 528"
                 transform="translate(845 293)"
                 fill="none"
                 stroke="#1a1818"
                 stroke-width="1"
              >
                <circle cx="12.511"
                        cy="12.511"
                        r="12.511"
                        stroke="none"
                >
                </circle>
                <circle cx="12.511"
                        cy="12.511"
                        r="12.011"
                        fill="none"
                >
                </circle>
              </g>
            </g>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        CAR
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="18.462"
               height="13.867"
               viewbox="0 0 18.462 13.867"
          >
            <path id="Parking"
                  d="M929.672,57.655a2.078,2.078,0,0,0-1.993,1.474l-.953,3.185-1.343-.455-.434,1.3,1.365.455-.434,1.474a.673.673,0,0,0-.022.195v5.547a.7.7,0,0,0,.693.693h2.08l.239-.693h10.617l.238.693h2.08a.7.7,0,0,0,.693-.693v-.585a.7.7,0,0,0,0-.109V65.282a.651.651,0,0,0-.022-.195l-.433-1.474,1.365-.455-.434-1.3-1.343.455-.953-3.185a2.078,2.078,0,0,0-1.994-1.474Zm0,1.387h9.014a.7.7,0,0,1,.672.5l.889,2.968H928.113L929,59.54A.7.7,0,0,1,929.672,59.042ZM927.7,63.9h12.958l.455,1.517v4.03H927.246v-4.03Zm1.278.693a1.04,1.04,0,1,0,1.04,1.04A1.04,1.04,0,0,0,928.979,64.589Zm10.4,0a1.04,1.04,0,1,0,1.04,1.04A1.04,1.04,0,0,0,939.38,64.589Zm-7.974,2.08-.866,2.08h1.5l.3-.693h3.683l.3.693h1.5l-.867-2.08Z"
                  transform="translate(-924.949 -57.655)"
            >
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        CAR_PARK
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="18.462"
               height="13.867"
               viewbox="0 0 18.462 13.867"
          >
            <path id="Parking"
                  d="M929.672,57.655a2.078,2.078,0,0,0-1.993,1.474l-.953,3.185-1.343-.455-.434,1.3,1.365.455-.434,1.474a.673.673,0,0,0-.022.195v5.547a.7.7,0,0,0,.693.693h2.08l.239-.693h10.617l.238.693h2.08a.7.7,0,0,0,.693-.693v-.585a.7.7,0,0,0,0-.109V65.282a.651.651,0,0,0-.022-.195l-.433-1.474,1.365-.455-.434-1.3-1.343.455-.953-3.185a2.078,2.078,0,0,0-1.994-1.474Zm0,1.387h9.014a.7.7,0,0,1,.672.5l.889,2.968H928.113L929,59.54A.7.7,0,0,1,929.672,59.042ZM927.7,63.9h12.958l.455,1.517v4.03H927.246v-4.03Zm1.278.693a1.04,1.04,0,1,0,1.04,1.04A1.04,1.04,0,0,0,928.979,64.589Zm10.4,0a1.04,1.04,0,1,0,1.04,1.04A1.04,1.04,0,0,0,939.38,64.589Zm-7.974,2.08-.866,2.08h1.5l.3-.693h3.683l.3.693h1.5l-.867-2.08Z"
                  transform="translate(-924.949 -57.655)"
            >
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        FERRY
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        GONDOLA
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        MICROMOBILITY
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        MICROMOBILITY_RENT
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        SCOOTER
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        RAIL
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="26.023"
               height="26.022"
               viewbox="0 0 26.023 26.022"
          >
            <g id="Group_57691"
               data-name="Group 57691"
               transform="translate(-703.5 -1378.855)"
            >
              <g id="Regional_Rail_Icon"
                 data-name="Regional Rail Icon"
                 transform="translate(709.744 1380.672)"
              >
                <path id="Path_36282"
                      data-name="Path 36282"
                      d="M50.43,217.073h7.787l-.275-.62-7.19,0Zm7.137-1.619-.174-.322h.99l.183.322h.656l.6.993h-.694l.363.62H60.2l.7,1.164h-.759l.629,1.1h-1.8l-.52-1.093-8.626.006-.555,1.087H47.491l.629-1.1h-.759l.688-1.164h.73l.354-.62h-.694l.6-.993h.656l.183-.322h.99l-.174.322Z"
                      transform="translate(-47.361 -199.074)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36283"
                      data-name="Path 36283"
                      d="M59.91,189.72h.815v-.5H61.2l1.1-2.369,0-7.063H50.611v7.078l1.1,2.346h.508v.5h.815v-.836l.414,0,.544-.871v-.857h.7l.032-6.36c.018-.213.139-.192.467-.219q.062,0,2.473,0c.357,0,.514-.038.514.29v6.292h.715l0,.863.555.871H59.9v.836Z"
                      transform="translate(-49.651 -174.173)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36284"
                      data-name="Path 36284"
                      d="M56.687,162.042v-.476l.851.027a4.842,4.842,0,0,1,1.533.417c.068.047.151.1.213.035.03-.035.366-.411.369-.414a2.852,2.852,0,0,0-.662-.626l-1.45-.233H55.275l-1.2.225a3.019,3.019,0,0,0-.653.662,3.414,3.414,0,0,0,.369.419c.006,0,.065-.03.18-.089a3.4,3.4,0,0,1,1.253-.375l.978-.044v.476h-.275v.806h.275v1.013l-2.9.222a4.42,4.42,0,0,0-2.334.809c-.322.3-.346,1.182-.36,1.182H62.3s-.024-.969-.428-1.22a6.517,6.517,0,0,0-2.659-.795l-2.526-.2v-1.013h.23v-.806h-.23Z"
                      transform="translate(-49.651 -160.772)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36285"
                      data-name="Path 36285"
                      d="M66.3,184.078a.475.475,0,0,1,.476-.476h1.291a.475.475,0,0,1,.476.476v1.391a.475.475,0,0,1-.476.476H66.777a.475.475,0,0,1-.476-.476v-1.391"
                      transform="translate(-60.706 -176.858)"
                      fill="#fff"
                      fill-rule="evenodd"
                >
                </path>
                <path id="Path_36286"
                      data-name="Path 36286"
                      d="M78.786,182.462h1.731a.154.154,0,0,1,.168.139v2.121a.184.184,0,0,1-.168.2H78.789a.186.186,0,0,1-.168-.2V182.6a.152.152,0,0,1,.165-.139"
                      transform="translate(-69.387 -176.055)"
                      fill="#4c748c"
                >
                </path>
                <path id="Path_36287"
                      data-name="Path 36287"
                      d="M54.676,182.392h1.731a.154.154,0,0,1,.168.139v2.121a.184.184,0,0,1-.168.2H54.679a.186.186,0,0,1-.168-.2v-2.121a.153.153,0,0,1,.165-.139"
                      transform="translate(-52.399 -176.005)"
                      fill="#4c748c"
                >
                </path>
                <path id="Path_36288"
                      data-name="Path 36288"
                      d="M58.361,197.123a.56.56,0,1,1,.561.561.561.561,0,0,1-.561-.561"
                      transform="translate(-55.112 -185.99)"
                      fill="#4c748c"
                      fill-rule="evenodd"
                >
                </path>
                <path id="Path_36289"
                      data-name="Path 36289"
                      d="M78.771,175.09H80.78s.012-.245.009-.452c0-.189-.006-.233-.192-.3a3.52,3.52,0,0,0-1.557-.272c-.139,0-.242.041-.269.133Z"
                      transform="translate(-69.492 -170.139)"
                      fill="#4c748c"
                >
                </path>
                <path id="Path_36290"
                      data-name="Path 36290"
                      d="M57.219,175.12H55.228s-.012-.242-.009-.452c0-.186.006-.233.192-.3a3.52,3.52,0,0,1,1.557-.272c.139,0,.242.041.269.133Z"
                      transform="translate(-52.898 -170.16)"
                      fill="#4c748c"
                >
                </path>
                <path id="Path_36291"
                      data-name="Path 36291"
                      d="M78.261,197.06a.56.56,0,1,1,.561.561.565.565,0,0,1-.561-.561"
                      transform="translate(-69.133 -185.947)"
                      fill="#4c748c"
                      fill-rule="evenodd"
                >
                </path>
                <rect id="Rectangle_1200"
                      data-name="Rectangle 1200"
                      width="4.446"
                      height="0.874"
                      transform="translate(4.564 13.187)"
                      fill="#fff"
                >
                </rect>
                <rect id="Rectangle_1201"
                      data-name="Rectangle 1201"
                      width="3.568"
                      height="1.176"
                      rx="0.588"
                      transform="translate(5.043 3.796)"
                      fill="#4c748c"
                >
                </rect>
                <path id="Path_36292"
                      data-name="Path 36292"
                      d="M65.551,174.708a.366.366,0,1,1,.366.366.366.366,0,0,1-.366-.366"
                      transform="translate(-60.178 -170.333)"
                      fill="#fff"
                      fill-rule="evenodd"
                >
                </path>
                <path id="Path_36293"
                      data-name="Path 36293"
                      d="M73.021,174.748a.366.366,0,1,1,.366.366.366.366,0,0,1-.366-.366"
                      transform="translate(-65.441 -170.362)"
                      fill="#fff"
                      fill-rule="evenodd"
                >
                </path>
                <path id="Path_36296"
                      data-name="Path 36296"
                      d="M50.43,217.073h7.787l-.275-.62-7.19,0Zm7.137-1.619-.174-.322h.99l.183.322h.656l.6.993h-.694l.363.62H60.2l.7,1.164h-.759l.629,1.1h-1.8l-.52-1.093-8.626.006-.555,1.087H47.491l.629-1.1h-.759l.688-1.164h.73l.354-.62h-.694l.6-.993h.656l.183-.322h.99l-.174.322Z"
                      transform="translate(-47.361 -199.074)"
                >
                </path>
                <path id="Path_36297"
                      data-name="Path 36297"
                      d="M59.91,189.72h.815v-.5H61.2l1.1-2.369,0-7.063H50.611v7.078l1.1,2.346h.508v.5h.815v-.836l.414,0,.544-.871v-.857h.7l.032-6.36c.018-.213.139-.192.467-.219q.062,0,2.473,0c.357,0,.514-.038.514.29v6.292h.715l0,.863.555.871H59.9v.836Z"
                      transform="translate(-49.651 -174.173)"
                >
                </path>
                <path id="Path_36298"
                      data-name="Path 36298"
                      d="M56.687,162.042v-.476l.851.027a4.842,4.842,0,0,1,1.533.417c.068.047.151.1.213.035.03-.035.366-.411.369-.414a2.852,2.852,0,0,0-.662-.626l-1.45-.233H55.275l-1.2.225a3.019,3.019,0,0,0-.653.662,3.414,3.414,0,0,0,.369.419c.006,0,.065-.03.18-.089a3.4,3.4,0,0,1,1.253-.375l.978-.044v.476h-.275v.806h.275v1.013l-2.9.222a4.42,4.42,0,0,0-2.334.809c-.322.3-.346,1.182-.36,1.182H62.3s-.024-.969-.428-1.22a6.517,6.517,0,0,0-2.659-.795l-2.526-.2v-1.013h.23v-.806h-.23Z"
                      transform="translate(-49.651 -160.772)"
                >
                </path>
                <path id="Path_36299"
                      data-name="Path 36299"
                      d="M66.3,184.078a.475.475,0,0,1,.476-.476h1.291a.475.475,0,0,1,.476.476v1.391a.475.475,0,0,1-.476.476H66.777a.475.475,0,0,1-.476-.476v-1.391"
                      transform="translate(-60.706 -176.858)"
                      fill-rule="evenodd"
                >
                </path>
                <path id="Path_36300"
                      data-name="Path 36300"
                      d="M78.786,182.462h1.731a.154.154,0,0,1,.168.139v2.121a.184.184,0,0,1-.168.2H78.789a.186.186,0,0,1-.168-.2V182.6a.152.152,0,0,1,.165-.139"
                      transform="translate(-69.387 -176.055)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36301"
                      data-name="Path 36301"
                      d="M54.676,182.392h1.731a.154.154,0,0,1,.168.139v2.121a.184.184,0,0,1-.168.2H54.679a.186.186,0,0,1-.168-.2v-2.121a.153.153,0,0,1,.165-.139"
                      transform="translate(-52.399 -176.005)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36302"
                      data-name="Path 36302"
                      d="M58.361,197.123a.56.56,0,1,1,.561.561.561.561,0,0,1-.561-.561"
                      transform="translate(-55.112 -185.99)"
                      fill="#fff"
                      fill-rule="evenodd"
                >
                </path>
                <path id="Path_36303"
                      data-name="Path 36303"
                      d="M78.771,175.09H80.78s.012-.245.009-.452c0-.189-.006-.233-.192-.3a3.52,3.52,0,0,0-1.557-.272c-.139,0-.242.041-.269.133Z"
                      transform="translate(-69.492 -170.139)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36304"
                      data-name="Path 36304"
                      d="M57.219,175.12H55.228s-.012-.242-.009-.452c0-.186.006-.233.192-.3a3.52,3.52,0,0,1,1.557-.272c.139,0,.242.041.269.133Z"
                      transform="translate(-52.898 -170.16)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36305"
                      data-name="Path 36305"
                      d="M78.261,197.06a.56.56,0,1,1,.561.561.565.565,0,0,1-.561-.561"
                      transform="translate(-69.133 -185.947)"
                      fill="#fff"
                      fill-rule="evenodd"
                >
                </path>
                <rect id="Rectangle_1205"
                      data-name="Rectangle 1205"
                      width="4.446"
                      height="0.874"
                      transform="translate(4.564 13.187)"
                >
                </rect>
                <path id="Path_36560"
                      data-name="Path 36560"
                      d="M.476,0H3.093a.476.476,0,0,1,.476.476V.7a.476.476,0,0,1-.476.476H.476A.476.476,0,0,1,0,.7V.476A.476.476,0,0,1,.476,0Z"
                      transform="translate(5.043 3.796)"
                      fill="#fff"
                >
                </path>
                <path id="Path_36306"
                      data-name="Path 36306"
                      d="M65.551,174.708a.366.366,0,1,1,.366.366.366.366,0,0,1-.366-.366"
                      transform="translate(-60.178 -170.333)"
                      fill-rule="evenodd"
                >
                </path>
                <path id="Path_36307"
                      data-name="Path 36307"
                      d="M73.021,174.748a.366.366,0,1,1,.366.366.366.366,0,0,1-.366-.366"
                      transform="translate(-65.441 -170.362)"
                      fill-rule="evenodd"
                >
                </path>
              </g>
              <g id="Group_57686"
                 data-name="Group 57686"
                 transform="translate(704 1379.355)"
              >
                <path id="Path_36325"
                      data-name="Path 36325"
                      d="M31.279,26.757A12.511,12.511,0,1,1,43.791,14.245,12.526,12.526,0,0,1,31.279,26.757"
                      transform="translate(-18.768 -1.734)"
                      fill="none"
                      stroke="#000"
                      stroke-width="1"
                >
                </path>
              </g>
            </g>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        STREETCAR
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="26.023"
               height="26.022"
               viewbox="0 0 26.023 26.022"
          >
            <g id="Group_57632"
               data-name="Group 57632"
               transform="translate(-299.5 -1876.595)"
            >
              <path id="Path_36325"
                    data-name="Path 36325"
                    d="M31.279,26.757A12.511,12.511,0,1,1,43.791,14.245,12.526,12.526,0,0,1,31.279,26.757"
                    transform="translate(281.232 1875.36)"
                    fill="none"
                    stroke="#1a1818"
                    stroke-width="1"
              >
              </path>
              <path id="Path_36326"
                    data-name="Path 36326"
                    d="M42.055,42.153l6.107-5.24H40.543l-5.4-5.409H35.14l0,0,0,0,5.209-5.239h6.8L41.047,31.5h7.619l5.4,5.41h.007l0,0,0,0L48.86,42.153ZM47.7,36.4l-4.217-4.217a.6.6,0,0,0-.428-.177H41.508l4.2,4.2a.671.671,0,0,0,.475.2Zm.483.492,0,0,0,0Z"
                    transform="translate(267.907 1855.4)"
              >
              </path>
            </g>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        SUBWAY
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="26.023"
               height="26.022"
               viewbox="0 0 26.023 26.022"
          >
            <g id="Group_57632"
               data-name="Group 57632"
               transform="translate(-299.5 -1876.595)"
            >
              <path id="Path_36325"
                    data-name="Path 36325"
                    d="M31.279,26.757A12.511,12.511,0,1,1,43.791,14.245,12.526,12.526,0,0,1,31.279,26.757"
                    transform="translate(281.232 1875.36)"
                    fill="none"
                    stroke="#1a1818"
                    stroke-width="1"
              >
              </path>
              <path id="Path_36326"
                    data-name="Path 36326"
                    d="M42.055,42.153l6.107-5.24H40.543l-5.4-5.409H35.14l0,0,0,0,5.209-5.239h6.8L41.047,31.5h7.619l5.4,5.41h.007l0,0,0,0L48.86,42.153ZM47.7,36.4l-4.217-4.217a.6.6,0,0,0-.428-.177H41.508l4.2,4.2a.671.671,0,0,0,.475.2Zm.483.492,0,0,0,0Z"
                    transform="translate(267.907 1855.4)"
              >
              </path>
            </g>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        TRAM
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="26.023"
               height="26.022"
               viewbox="0 0 26.023 26.022"
          >
            <g id="Group_57632"
               data-name="Group 57632"
               transform="translate(-299.5 -1876.595)"
            >
              <path id="Path_36325"
                    data-name="Path 36325"
                    d="M31.279,26.757A12.511,12.511,0,1,1,43.791,14.245,12.526,12.526,0,0,1,31.279,26.757"
                    transform="translate(281.232 1875.36)"
                    fill="none"
                    stroke="#1a1818"
                    stroke-width="1"
              >
              </path>
              <path id="Path_36326"
                    data-name="Path 36326"
                    d="M42.055,42.153l6.107-5.24H40.543l-5.4-5.409H35.14l0,0,0,0,5.209-5.239h6.8L41.047,31.5h7.619l5.4,5.41h.007l0,0,0,0L48.86,42.153ZM47.7,36.4l-4.217-4.217a.6.6,0,0,0-.428-.177H41.508l4.2,4.2a.671.671,0,0,0,.475.2Zm.483.492,0,0,0,0Z"
                    transform="translate(267.907 1855.4)"
              >
              </path>
            </g>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        TRANSIT
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="26.023"
               height="26.022"
               viewbox="0 0 26.023 26.022"
          >
            <g id="Group_57632"
               data-name="Group 57632"
               transform="translate(-299.5 -1876.595)"
            >
              <path id="Path_36325"
                    data-name="Path 36325"
                    d="M31.279,26.757A12.511,12.511,0,1,1,43.791,14.245,12.526,12.526,0,0,1,31.279,26.757"
                    transform="translate(281.232 1875.36)"
                    fill="none"
                    stroke="#1a1818"
                    stroke-width="1"
              >
              </path>
              <path id="Path_36326"
                    data-name="Path 36326"
                    d="M42.055,42.153l6.107-5.24H40.543l-5.4-5.409H35.14l0,0,0,0,5.209-5.239h6.8L41.047,31.5h7.619l5.4,5.41h.007l0,0,0,0L48.86,42.153ZM47.7,36.4l-4.217-4.217a.6.6,0,0,0-.428-.177H41.508l4.2,4.2a.671.671,0,0,0,.475.2Zm.483.492,0,0,0,0Z"
                    transform="translate(267.907 1855.4)"
              >
              </path>
            </g>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        WALK
      </td>
      <td>
        <div>
          <svg xmlns="http://www.w3.org/2000/svg"
               width="16"
               height="30"
               viewbox="0 0 9 15.162"
          >
            <path id="Icon_awesome-walking"
                  data-name="Icon awesome-walking"
                  d="M6.16,2.843A1.421,1.421,0,1,0,4.739,1.421,1.422,1.422,0,0,0,6.16,2.843Zm2.8,4.415-.69-.349-.287-.871A3.228,3.228,0,0,0,4.955,3.79a6.511,6.511,0,0,0-2.763.746A2.824,2.824,0,0,0,.721,5.9l-.2.4a.951.951,0,0,0,.42,1.27A.935.935,0,0,0,2.2,7.154l.2-.4A.941.941,0,0,1,2.888,6.3l.794-.32-.45,1.8a1.9,1.9,0,0,0,.441,1.741L5.446,11.45a1.911,1.911,0,0,1,.441.82l.542,2.17a.948.948,0,1,0,1.839-.459l-.657-2.635a1.894,1.894,0,0,0-.441-.82L5.822,9.055,6.332,7.02l.163.489a1.888,1.888,0,0,0,.939,1.1l.69.349a.935.935,0,0,0,1.258-.423.959.959,0,0,0-.423-1.273ZM2.18,11.423a1.837,1.837,0,0,1-.42.637L.279,13.544a.947.947,0,0,0,1.338,1.341l1.759-1.759a1.925,1.925,0,0,0,.42-.637l.4-1C2.559,9.7,3.051,10.251,2.793,9.9L2.18,11.423Z"
            >
            </path>
          </svg>
        </div>
      </td>
    </tr>
    <tr>
      <td>
        NONE_OF_THE_ABOVE
      </td>
      <td>
        <div>
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;
